rest-service/manager_rest/rest/resources_v3/nodes.py

```diff
@@ -14,25 +14,26 @@
 #  * limitations under the License.
 
 from manager_rest.security.authorization import authorize
-from manager_rest.dsl_functions import evaluate_intrinsic_functions
+from manager_rest.dsl_functions import (
+    evaluate_node,
+    evaluate_node_instance
+)
 
 from .. import rest_decorators
 from ..resources_v1.nodes import NodeInstancesId as v1_NodeInstancesId
 from ..resources_v2 import Nodes as v2_Nodes
 
 
 class Nodes(v2_Nodes):
-    @rest_decorators.exceptions_handled
```

Comment:
Why is this needed?

Reply:
Committed by accident

our `rest_decorators.exceptions_handled` is absolutely fucked and you need to remove it if you want to see exceptions

made CY-1687 about it